wiki.md

```diff
@@ -1,22 +1,22 @@
-<strong>Building Android</strong>
+## Building Android
```

Comment:
Id say this line is the same as the next. Append from start to finish here and nuke the next line.

Don't say (wip)